pyintelowl/cli/tags.py

```diff
@@ -31,6 +32,45 @@ def ls(ctx: ClickContext, as_json: bool):
         ctx.obj.logger.fatal(str(e))
 
 
+@tags.command(help="Create new Tag")
+@click.argument("label", type=str)
+@click.argument("color", type=str)
+@add_options(json_flag_option)
+@click.pass_context
+def new(ctx: ClickContext, as_json: bool, color: str, label: str):
+    ctx.obj.logger.info("Adding new Tag..")
+    try:
+        ans = ctx.obj.create_tag(label, color)
+        if as_json:
+            rprint(json.dumps(ans, indent=4))
+        else:
+            ans = [ans]
+            _print_tags_table(ctx.obj.logger, ans)
+    except IntelOwlClientException as e:
+        print(e.response.status_code)
```

Comment:
status code is displayed with the error anyway so not required. Same goes for `edit` function.